be/src/controlers/rest.router.ts

```diff
@@ -64,7 +67,11 @@ router.get('/vocab/:id/entities', (req: Request, res: Response) => {
 });
 
 router.post('/vocab/:id/entities', (req: Request, res: Response) => {
-  res.json({statusCode:206});
+    const vocabID: string = req.params.id;
```

Comment:
Check if the ID is falsy, if so return a 404. You can copy some logic from:
https://github.com/TailoredMediaProject/tm-knowledge-store/pull/17/files#diff-9af31357cfba9bb58fc7832c1d73504fdfc6e074994311a3e09a24aecc027b59R57